qemu/tests/cfg/blockdev_inc_backup_after_commit.cfg

```diff
@@ -40,7 +40,10 @@
     gluster_direct:
         enable_gluster_data = yes
     ceph:
-        enable_ceph_data = yes
+        enable_ceph_base = no
+        enable_ceph_inc = no
+        enable_ceph_datasn = no
```

Comment:
Remove these 3 lines for we defined them below

Reply:
Have updated it, thanks zhenchao.